source/common/network/utility.h

```diff
@@ -201,6 +201,9 @@ class Utility {
    */
   static absl::uint128 Ip6htonl(const absl::uint128& address);
 
+  static Address::InstanceConstSharedPtr
+  parseProtobufAddress(const envoy::api::v2::core::Address& proto_address);
```

Comment:
Maybe rename to `protobufAddressToAddress` for symmetry?

Reply:
Done.